mailer.rst

```diff
@@ -0,0 +1,649 @@
+Sending Emails with Mailer
+==========================
+
+.. versionadded:: 4.3
+    The Mailer component was added in Symfony 4.3. The previous solution - Swift Mailer -
+    is still valid: :doc:`Swift Mailer</email>`.
+
+Installation
+------------
+
+.. caution::
+
+    The Mailer component is experimental in Symfony 4.3: some backwards compatibility
+    breaks could occur before 4.4.
+
+Symfony's Mailer & :doc:`Mime </components/mime>` components form a *powerful* system
+for creating and sending emails - complete with support for multipart messages, Twig
+integration, CSS inlining, file attachments and a lot more. Get them installed with:
+
+.. code-block:: terminal
+
+    $ composer require symfony/mailer
+
+Transport Setup
+---------------
+
+Emails are delivered via a "transport". And without installing anything else, you
+delivery emails over ``smtp`` by configuring your ``.env`` file:
+
+.. code-block:: bash
+
+    # .env
+    MAILER_DSN=smtp://user:pass@smtp.example.com
+
+Using a 3rd Party Transport
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+But an easier option is to send emails via a 3rd party provider. Mailer supports
+several - install whichever you want:
+
+==================  =============================================
+Service             Install with
+==================  =============================================
+Amazon SES          ``composer require symfony/amazon-mailer``
```

Comment:
the order is by popularity or random?

Reply:
I did it alphabetically, but put gmail last, as that's not really a great option